packages/@react-spectrum/list/src/RootDropIndicator.tsx

```diff
@@ -0,0 +1,25 @@
+import {ListViewContext} from './ListView';
+import React, {useContext, useRef} from 'react';
+import {useVisuallyHidden} from '@react-aria/visually-hidden';
+
+export default function RootDropIndicator() {
+  let {dropState, dropHooks} = useContext(ListViewContext);
+  let dropRef = useRef();
+  let {dropIndicatorProps} = dropHooks.useDropIndicator({
+    target: {type: 'root'}
+  }, dropState, dropRef);
+
+  let {visuallyHiddenProps} = useVisuallyHidden();
+  if (dropIndicatorProps['aria-hidden']) {
+    return null;
+  }
+
+  return (
+    <div
+      role="option"
```

Comment:
role="option" shouldn't be within a grid. This should be a row/gridcell combo like the other insertion indicators. And I suspect a button inside as well otherwise won't be able to reach this with VO on iOS.